### PR TITLE
rename `t.ok()` to `t.truthy()` and `t.notOk()` to `t.falsy()`

### PR DESCRIPTION
**Pull request by kentcdodds**
Pretty confident that this is ready to go.

sloth-sparkle-FFV4J8uEtJKGQ

### CODE REVIEW

File: test/fixture/power-assert.js [hunk 1]
```diff
@@ -3,11 +3,11 @@ import test from '../../';
...
 
-	t.ok(a === 'bar');
+	t.truthy(a === 'bar');
 });
 
 test.serial(t => {
 	const a = 'bar';
 
-	t.ok(a === 'foo', 'with message');
+	t.truthy(a === 'foo', 'with message');
```
Comment (sindresorhus) [inline test/fixture/power-assert.js]: Actually, these should just be `t.true`.

File: test/fixture/one-pass-one-fail.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import test from '../../';
 
 test('this is a passing test', t => {
-	t.ok(true);
+	t.truthy(true);
```
Comment (sindresorhus) [inline test/fixture/one-pass-one-fail.js]: `t.pass()`

File: test/fixture/one-pass-one-fail.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 });
 
 test('this is a failing test', t => {
-	t.ok(false);
+	t.truthy(false);
```
Comment (sindresorhus) [inline test/fixture/one-pass-one-fail.js]: `t.fail();`

File: test/fixture/slow-exit.js [hunk 4]
```diff
@@ -20,7 +20,7 @@ test.cb('long running', t => {
 	}, {alwaysLast: true});
 
 	setTimeout(() => {
-		t.ok(true);
+		t.truthy(true);
```
Comment (sindresorhus) [inline test/fixture/slow-exit.js]: `t.pass();`

File: readme.md [hunk 5]
```diff
@@ -843,14 +843,14 @@ test(t => {
 	const a = /foo/;
 	const b = 'bar';
 	const c = 'baz';
-	t.ok(a.test(b) || b === c);
+	t.truthy(a.test(b) || b === c);
```
Comment (sindresorhus) [inline readme.md]: `t.true()`

File: readme.md [hunk 6]
```diff
@@ -843,14 +843,14 @@ test(t => {
...
 });
 ```
 
 Will output:
 
 ```
-t.ok(a.test(b) || b === c)
+t.truthy(a.test(b) || b === c)
```
Comment (sindresorhus) [inline readme.md]: `t.true()`

File: test/fixture/one-pass-one-fail.js (hunk 3 above)
Comment (kentcdodds) [inline test/fixture/one-pass-one-fail.js]: I was going to go through and update some of these to a more sensible test, but decided to opt for a direct refactor. I'll go through and update these to be better :+1:

**Comment by kentcdodds** [comment]
Responded to all comments :-)

**Comment by sindresorhus** [comment]
LGTM

File: readme.md [hunk 7]
```diff
@@ -843,14 +843,14 @@ test(t => {
...
-	t.ok(a.test(b) || b === c);
+	t.true(a.test(b) || b === c);
 });
 ```
 
 Will output:
 
 ```
-t.ok(a.test(b) || b === c)
+t.true(a.test(b) || b === c)
```
Comment (jfmengels) [inline readme.md]: The power-assert example right under this should be moved a bit to the right. The lines do not point to what is described.

File: readme.md [hunk 8]
```diff
@@ -843,14 +843,14 @@ test(t => {
...
        |    |     |     |
```
Comment (forresst) [inline readme.md]: add two spaces to align

Comment (kentcdodds) [inline readme.md]: Updated. Thanks @jfmengels and @forresst

**Comment by sindresorhus** [comment]
Seems you need to update some test assertions too.

**Comment by kentcdodds** [comment]
I guess this is why we have Travis :-)

warnerarchive-classic-film-alfred-hitchcock-suspicion-l2JJFTildjINz4cKY

**Comment by kentcdodds** [comment]
Updated. Should be good to go now :D

**Comment by vdemedes** [comment]
LGTM!

**Pull request merged**